OrbitSsh/ContextTests.cpp

```diff
@@ -12,11 +12,11 @@ namespace OrbitSsh {
 
 TEST(Context, Create) {
   auto context = Context::Create();
-  ASSERT_TRUE(context.has_value());
-  ASSERT_TRUE(context.value().active());
+  ASSERT_TRUE(context);
+  ASSERT_TRUE(context.value().Active());
 
   auto context2 = std::move(context.value());
-  ASSERT_FALSE(context.value().active());
-  ASSERT_TRUE(context2.active());
+  ASSERT_FALSE(context.value().Active());
```

Comment:
this is trivial getter - the reason it was named with active()

Reply:
Did we decide on doing that? There are several situations related to protos where this also comes up

Reply:
I found this is the style guide:
> Regular functions have mixed case; accessors and mutators may be named like variables.
so I guess will change it back to `active()`

Reply:
I am fine both ways, my personal preference is to use them for trivial accessors. Since style guide says may - you can do it one way or the other.

Reply:
done